build/develop-env/salobj/Dockerfile

```diff
@@ -79,19 +79,21 @@ RUN source /home/saluser/.setup_salobj.sh && \
 
 WORKDIR /home/saluser/repos/ts_utils
 RUN source /home/saluser/.setup_salobj.sh && \
-    setup ts_utils -t current && \
-    scons
+    python -m pip install --no-deps --ignore-installed .
 
 ARG ts_dds
 
 WORKDIR /home/saluser/repos/ts_salobj
 RUN source /home/saluser/.setup_salobj.sh && \
     mamba install -c lsstts -y ts-dds=${ts_dds} && \
+    setup ts_config_ocs -t current && \
+    setup ts_xml -t current && \
+    setup ts_ddsconfig -t current && \
     setup ts_sal -t current && \
     setup ts_idl -t current && \
     export LSST_DDS_IP=`ip route get 1 | awk '{print $7;exit}'` && \
-    setup ts_salobj -t current && \
-    scons || pytest --color=no --lf
+    python -m pip install --no-deps --ignore-installed . && \
```

Comment:
If you adopt my suggestion to `pip install` the package in a separate earlier step, you won't need to do it here.